### PR TITLE
DEV: DAQ task, flt pnt, RIX hutch

### PR DESCRIPTION
**Pull request by sfsyunusSLAC**
- Reenabled DAQ task -- Zach
- Unchecked Floating point exceptions in PlcTask -- Maggie
- Added additional I/O for RIX hutch

### CODE REVIEW

**Comment by sfsyunusSLAC** [comment]
Also updated IP address and AMS net IDs on TC side, still need to update IOC

File: lcls-plc-rixs-optics/rixs_optics/rixs_optics.plcproj [hunk 1]
```diff
@@ -173,25 +169,34 @@
     </PlaceholderReference>
   </ItemGroup>
   <ItemGroup>
-    <LibraryReference Include="LCLS General,newest,SLAC">
+    <LibraryReference Include="LCLS General,2.4.2,SLAC">
```
Comment (ZLLentz) [inline lcls-plc-rixs-optics/rixs_optics/rixs_optics.plcproj:172]: Thank you for taking these off of newest!

File: lcls-plc-rixs-optics/lcls-plc-rixs-optics.tsproj [hunk 2]
```diff
@@ -25,7 +25,7 @@
 				<Task Id="7" Priority="1" CycleTime="20000" AmsPort="352" AdtTasks="true">
 					<Name>SerialIO</Name>
 				</Task>
-				<Task Id="8" Disabled="true" Priority="19" CycleTime="10000" AmsPort="353" AdtTasks="true">
+				<Task Id="8" Priority="19" CycleTime="10000" AmsPort="353" AdtTasks="true">
```
Comment (ZLLentz) [inline lcls-plc-rixs-optics/lcls-plc-rixs-optics.tsproj:28]: Confirming that this is the part that I needed for the daq task :+1:

Comment (ZLLentz) [reply lcls-plc-rixs-optics/lcls-plc-rixs-optics.tsproj:28]: Obsoletes #49

**Review by ZLLentz — APPROVED** [review]
Seeing a review request with 116 file changes gave me a heart attack until I realized it was just hundreds of additions to the IO bus.

I went through the GVL line by line and the pragmas all look correct to me.

File: lcls-plc-rixs-optics/rixs_optics/rixs_optics.plcproj (hunk 1 above)
Comment (sfsyunusSLAC) [reply lcls-plc-rixs-optics/rixs_optics/rixs_optics.plcproj:172]: I have finally learned how to do this properly after years thanks to @ghalym

**Comment by ghalym** [comment]
The pragma linter failed!

**Review by ghalym — APPROVED** [review]

File: lcls-plc-rixs-optics/lcls-plc-rixs-optics.tsproj [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 					<TargetSelect TargetId="2">{BCA6EE0A-9CE1-4D3F-98CA-413ABC0D94FD}</TargetSelect>
+					<TargetSelect TargetId="2">{66689887-CCBD-452C-AC9A-039D997C6E66}</TargetSelect>
 					<TargetSelect TargetId="2">{BF78CFC7-2E63-42C3-8C07-BB6C346BFB8B}</TargetSelect>
 					<TargetSelect TargetId="2">{777F1598-583B-4503-99BB-7C02E0ABD97E}</TargetSelect>
 					<LicenseDevice DongleHardwareId="2" DongleDevice="#x03020003" DongleSystemId="{2BDA9270-9FD1-1DDB-58F3-2743C1D0CE01}"/>
 				</Target>
 			</Licenses>
 			<Tasks>
-				<Task Id="3" Priority="20" CycleTime="100000" AmsPort="350" AdtTasks="true">
+				<Task Id="3" Priority="20" CycleTime="100000" AmsPort="350" DisableFpExceptions="true" AdtTasks="true">
```
Comment (ghalym) [inline lcls-plc-rixs-optics/lcls-plc-rixs-optics.tsproj:19]: 👍

**Review by ghalym — APPROVED** [review]

**Comment by sfsyunusSLAC** [comment]
Pragma linter needs investigation... but turning this over to Maggie for now

**Pull request merged**